### ISSUE TITLE
unknown alias not reported as syntax error

### ISSUE THREAD

**Issue by larshp** [abaplint/abaplint#3095]
```abap
CLASS lcl DEFINITION.
  PUBLIC SECTION.
    ALIASES blah FOR zlif~blah.
ENDCLASS.

CLASS lcl IMPLEMENTATION.
ENDCLASS.
```

**Comment by marcellourbani** [comment]
Trying to write a rule for this

**Comment by larshp** [comment]
this should not be a separate rule

**Comment by marcellourbani** [comment]
where should it go then? implement_methods?
